doc/api/errors.md

```diff
@@ -1912,6 +1907,14 @@ added: v9.0.0
 removed: v10.0.0
 -->
 
+<a id="ERR_INDEX_OUT_OF_RANGE"></a>
```

Comment:
It seems this section is now inserted inside `ERR_HTTP_INVALID_CHAR` section,

Reply:
Whoops! copy/paste error. Fixed. Thanks.